cypress/integration/table/search.spec.ts

```diff
@@ -746,6 +746,21 @@ describe('Search', () => {
       );
     });
 
+    it('select a experiment Id and it appears in the experiment box', () => {
+      // experiment box
+      cy.get('[aria-label="open experiment search box"]')
```

Comment:
Could we try and use `getByRole`/other RTL `getBy` commands instead of native cypress `get` commands? Just reads nicer and harmonises the assertions across Jest, Cypress & Playwright a little